docs/sources/permissions/dashboard_folder_permissions.md

```diff
@@ -25,11 +25,30 @@ Permission levels:
 - **Edit**: Can edit and create dashboards. **Cannot** edit folder/dashboard permissions, or add, edit, or delete folders.
 - **View**: Can only view existing dashboards/folders.
 
+## Grant folder permissions
+
+1. In the sidebar, hover your mouse over the **Dashboards** (squares) icon and then click **Manage**.
+1. Hover your mouse cursor over a folder and click the gear icon to the right.
```

Comment:
At least on master, this seems incorrect. No gear icon, rather "go to folder"

![image](https://user-images.githubusercontent.com/1692624/95599368-fa536c80-0a1e-11eb-9613-e77c39811eb5.png)

Reply:
Nice catch! Updated.